metadata-ingestion/src/datahub/ingestion/source/confluent_schema_registry.py

```diff
@@ -346,22 +361,27 @@ def _get_schema_fields(
             )
         elif not self.source_config.ignore_warnings_on_schema_type:
             self.report.report_warning(
-                topic,
+                dataset_subtype,
                 f"Parsing kafka schema type {schema.schema_type} is currently not implemented",
             )
         return fields
 
     def _get_schema_metadata(
-        self, topic: str, platform_urn: str
+        self, dataset_subtype: str, platform_urn: str, is_subject: bool
     ) -> Optional[SchemaMetadata]:
+
         # Process the value schema
         schema, fields = self._get_schema_and_fields(
-            topic=topic, is_key_schema=False
+            dataset_subtype=dataset_subtype,
```

Comment:
can we keep topic name as it was before?

Reply:
or, as I wrote, maybe we should call to `subject` or `schema_name` ?

Reply:
Hi @treff7es based on the caller of get_schema_metadata() from kafka.py this can either be topic or schema, can we have kafka_entity instead of topic or subject which can be a common name?

caller from kafka.py
# 1. Create schemaMetadata aspect (pass control to SchemaRegistry)
  schema_metadata = self.schema_registry_client.get_schema_metadata(
      kafka_entity, platform_urn, is_subject
  )

Reply:
will change all occurrence of dataset_subtype to topic, using boolean is_subject we can identify if its topic or subject.